File: game-server/src/main/java/org/forfun/mmorpg/game/gm/GmDoc.java
package org.forfun.mmorpg.game.gm;

public class GmDoc {

    public static String printDoc() {
        StringBuilder sb = new StringBuilder();
        sb.append("--------gm命令说明-----------\n");
        for (GmCommands cmd : GmCommands.values()) {
            sb.append(cmd.getDesc() + "\t-->\t(" + cmd.getExample()).append(")\n");
        }

        return sb.toString();
    }

    public static void main(String[] args) {
        String doc = printDoc();
        System.out.println(doc);
    }
}

File: game-server/src/main/java/org/forfun/mmorpg/game/gm/GmDoc.kt
package org.forfun.mmorpg.game.gm

object GmDoc {

    fun printDoc(): String {
        val sb = StringBuilder()
        sb.append("--------gm命令说明-----------\n")
        for (cmd in GmCommands.entries) {
            sb.append(cmd.desc + "\t-->\t(" + cmd.example).append(")\n")
        }

        return sb.toString()
    }

    @JvmStatic
    fun main(args: Array<String>) {
        val doc = printDoc()
        println(doc)
    }
}
